From 7dc74d2a660f5deb8acc1d67039cc38c52f4ccd7 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Mon, 19 Apr 2021 23:55:12 -0300
Subject: [PATCH 1/2] rpmbuild bb shortcircuit

---
 build/pack.c | 6 +++---
 rpmbuild.c   | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/build/pack.c b/build/pack.c
index 1348b5f..7c10584 100644
--- a/build/pack.c
+++ b/build/pack.c
@@ -672,9 +672,9 @@ rpmRC packageBinaries(rpmSpec spec, const char *cookie, int cheating)
 	    headerPutBin(pkg->header, RPMTAG_SOURCEPKGID, spec->sourcePkgId,16);
 	}
 
-	if (cheating) {
-	    (void) rpmlibNeedsFeature(pkg, "ShortCircuited", "4.9.0-1");
-	}
+// 	if (cheating) {
+// 	    (void) rpmlibNeedsFeature(pkg, "ShortCircuited", "4.9.0-1");
+// 	}
 	
 	{   char *binFormat = rpmGetPath("%{_rpmfilename}", NULL);
 	    char *binRpm, *binDir;
diff --git a/rpmbuild.c b/rpmbuild.c
index ffb61fc..dc9410e 100644
--- a/rpmbuild.c
+++ b/rpmbuild.c
@@ -639,11 +639,11 @@ int main(int argc, char *argv[])
     case 'i':
 	ba->buildAmount |= RPMBUILD_INSTALL;
 	ba->buildAmount |= RPMBUILD_CHECK;
-	if ((buildChar == 'i') && shortCircuit)
+	if (/*(buildChar == 'i') && */ shortCircuit)
 	    break;
     case 'c':
 	ba->buildAmount |= RPMBUILD_BUILD;
-	if ((buildChar == 'c') && shortCircuit)
+	if (/*(buildChar == 'c') && */ shortCircuit)
 	    break;
     case 'p':
 	ba->buildAmount |= RPMBUILD_PREP;
-- 
2.31.1

