From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michael Ikey Doherty <michael.i.doherty@intel.com>
Date: Thu, 10 Apr 2014 12:11:24 +0100
Subject: [PATCH] Ensure "lib" is used and not "lib64"

Signed-off-by: Michael Ikey Doherty <michael.i.doherty@intel.com>
---
 installplatform | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/installplatform b/installplatform
index 12339fc..6bc7b9d 100755
--- a/installplatform
+++ b/installplatform
@@ -190,9 +190,10 @@ for ARCH in noarch `grep ^arch_canon $RPMRC | cut -d: -f2`; do
   # skip architectures for which we dont have full config parameters
   [ -z "$CANONARCH" ] && continue
 
-  if [ "$OS" = "linux" ] && [ "$CANONCOLOR" = 3 ]; then
-      LIB=${LIB}64
-  fi
+  # Enforce "lib" not "lib64" as library directory/name
+  #if [ "$OS" = "linux" ] && [ "$CANONCOLOR" = 3 ]; then
+  #    LIB=${LIB}64
+  #fi
 
   PPD="${DESTDIR}/${platformdir}/${ARCH}-${OS}"
   [ -d $PPD ] || mkdir -p $PPD
