From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Fri, 27 Oct 2017 15:41:28 +0000
Subject: [PATCH] add an fflush

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 build/files.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/files.c b/build/files.c
index ca3e4a9..2de6c9b 100644
--- a/build/files.c
+++ b/build/files.c
@@ -2736,6 +2736,7 @@ static int checkFiles(const char *buildRoot, StringBuf fileList)
 	    rc = (_unpackaged_files_terminate_build) ? 1 : 0;
 	    rpmlog((rc ? RPMLOG_ERR : RPMLOG_WARNING),
 		_("Installed (but unpackaged) file(s) found:\n%s"), t);
+	    fflush(stdout);
 	}
     }
     
