From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Leandro Dorileo <leandro.maciel.dorileo@intel.com>
Date: Thu, 19 Sep 2019 12:37:00 -0700
Subject: [PATCH] discover uid0 based on /usr/share/defaults

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 34ea85f..0dcb906 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1003,8 +1003,8 @@ if test "$with_dmalloc" = yes ; then
   LIBS="$LIBS -ldmalloc"
 fi
 
-user_with_uid0=$(awk -F: '$3==0 {print $1;exit}' /etc/passwd)
-group_with_gid0=$(awk -F: '$3==0 {print $1;exit}' /etc/group)
+user_with_uid0=$(awk -F: '$3==0 {print $1;exit}' /usr/share/defaults/etc/passwd)
+group_with_gid0=$(awk -F: '$3==0 {print $1;exit}' /usr/share/defaults/etc/group)
 AC_DEFINE_UNQUOTED([UID_0_USER],["$user_with_uid0"],[Get the user name having userid 0])
 AC_DEFINE_UNQUOTED([GID_0_GROUP],["$group_with_gid0"],[Get the group name having groupid 0])
 
